docs/docs/alephium/entrypoints.md

```diff
@@ -0,0 +1,652 @@
+| -------------------- | --------- | --------------------------------------------------------------------- |
+| poolKey             | PoolKey   | The pool key for which you want to create a position.                 |
+| lowerTick           | I256       | The lower tick index of your position.                                      |
+| upperTick           | I256       | The upper tick index of your position.                                      |
+| liquidityDelta      | U256 | The liquidity you want to provide.                                    |
+| slippageLimitLower | U256 | The lower square root of price fluctuation you are willing to accept. |
+| slippageLimitUpper | U256 | The upper square root of price fluctuation you are willing to accept. |
+
+
+#### Errors
+
+| Code                            | Description                                                                                |
+| ------------------------------- | ------------------------------------------------------------------------------------------ |
+| `ZeroLiquidity`                 | Fails if the user attempts to open a position with zero liquidity.                         |
+| `InvalidTickIndex` | Fails if the user attempts to create a position with invalid tick indexes. |
+| `InvalidTickSpacing` | Fails if the user attempts to create a position with invalid tick spacing. |
+| `PriceLimitReached`             | Fails if the price has reached the slippage limit.                                         |
+| `PoolNotFound`                  | Fails if pool does not exist.                                                              |
+
+### Transfer position
+
+```rust
+@using(preapprovedAssets = true, checkExternalCaller = false)
+pub fn transferPosition(index: U256, newOwner: Address) -> ()
+```
+
+This function changes ownership of an existing position based on the position index in the user's position list. You can only change ownership of positions that you own; otherwise, it will return an error.
+
+#### Input parameters
+
+| Name     | Type      | Description                                        |
+| -------- | --------- | -------------------------------------------------- |
+| index    | U256       | Index of the position in the user's position list. |
+| newOwner | Address | Address of the user who will receive the position. |
+
+#### Errors
+
+| Code                            | Description                                                                                |
+| ------------------------------- | ------------------------------------------------------------------------------------------ |
+| `PositionNotFound`                 | Fails if position does not exist.                        |
+
+### Remove position
+
+```rust
+@using(checkExternalCaller = false)
+pub fn removePosition(index: U256) -> () 
+```
+
+This function removes a position from the user's position list and transfers the tokens used to create the position to the user's address.
```

Comment:
It is worth adding information that all unclaimed fees generated by this position will also be transferred upon removing the position